src/agent/onefuzz-agent/src/tasks/fuzz/libfuzzer_fuzz.rs

```diff
@@ -370,35 +392,98 @@ impl Timer {
 // running worker runs, this can result in misleading gaps and binning artifacts. In effect, we
 // are approximating nearest-neighbor interpolation on the runtime stats time series.
 async fn report_runtime_stats(
-    workers: usize,
     mut stats_channel: mpsc::UnboundedReceiver<RuntimeStats>,
     heartbeat_client: impl HeartbeatSender,
 ) -> Result<()> {
     // Cache the last-reported stats for a given worker.
     //
     // When logging stats, the most recently reported runtime stats will be used for any
     // missing data. For time-triggered logging, it will be used for all workers.
-    let mut last_reported: Vec<Option<RuntimeStats>> =
-        std::iter::repeat(None).take(workers).collect();
+    let mut total = TotalStats::default();
+
+    // report all zeros to start
+    total.report();
 
     let timer = Timer::new(RUNTIME_STATS_PERIOD);
 
     loop {
         tokio::select! {
```

Comment:
Not a blocking comment but i noticed that this pattern seems a bit slow when dequeuing a lot of messages. 
i have been using try_recv instead for polling at a regular interval
``` Rust
match stats_channel.try_recv() {
        Ok(stats) => {
            heartbeat_client.alive();
            total.update(stats);
            total.report();
        }
        Err(mpsc::error::TryRecvError::Empty) => {
           total.report();
           delay_for(RUNTIME_STATS_PERIOD).await,
        }
        Err(mpsc::error::TryRecvError::Closed) => break,
    }
```
I don't think this would b a big deal for this PR though